plugins/woocommerce/client/admin/client/settings-payments/settings-payments-bacs.tsx

```diff
@@ -0,0 +1,182 @@
+/**
+ * External dependencies
+ */
+import {
+	Card,
+	CardBody,
+	CheckboxControl,
+	TextControl,
+	TextareaControl,
+	Button,
+} from '@wordpress/components';
+import { useState, useEffect } from 'react';
+import { __ } from '@wordpress/i18n';
+import { useDispatch, useSelect } from '@wordpress/data';
+import {
+	PAYMENT_GATEWAYS_STORE_NAME,
```

Comment:
We should import `paymentGatewaysStore` and use it like `select( paymentGatewaysStore )` so that types are automatically available.

The `PAYMENT_GATEWAYS_STORE_NAME` only exists for backward compatibility. 🙂